### PR TITLE
Register page

### PR DESCRIPTION
**Pull request by Sanduntha**

### CODE REVIEW

File: src/app/App.jsx [hunk 1]
```diff
@@ -1,11 +1,12 @@
+import RegisterPage from '../pages/RegisterPage/RegisterPage'
 import './App.css'
 
 function App() {
```
Comment (rush144) [inline src/app/App.jsx]: remove unused lines

File: src/pages/RegisterPage/RegisterPage.jsx [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+    }
+
+    console.log("Form Data:", formData);
+    alert("Registration successful!");
+  };
+
+  return (
+
+    <div className="box">
+
```
Comment (rush144) [inline src/pages/RegisterPage/RegisterPage.jsx]: remove unused lines

File: src/app/App.jsx (hunk 1 above)
Comment (Sanduntha) [reply src/app/App.jsx]: Done

File: src/pages/RegisterPage/RegisterPage.jsx (hunk 2 above)
Comment (Sanduntha) [reply src/pages/RegisterPage/RegisterPage.jsx]: Done

**Pull request merged**